test/over_react/component_declaration/builder_integration_tests/component2/required_accessor_integration_test.dart

```diff
@@ -11,86 +11,66 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-import 'dart:js';
 
 import 'package:over_react/over_react.dart';
+import 'package:over_react_test/over_react_test.dart';
 import 'package:test/test.dart';
-import 'package:react/react_client/react_interop.dart' show PropTypes;
 
 import '../../../../test_util/test_util.dart';
 
 part 'required_accessor_integration_test.over_react.g.dart';
```

Comment:
Maybe I missed it - what is the fundamental difference between these tests - and the ones within `constant_required_accessor_integration_test.dart`?

Reply:
Really good question. The main difference is in the annotation of the component being tested. One uses `@requiredProp` / `@nullableRequiredProp` and the other uses `@Accessor`. The tests are functionally the same though.